### PR TITLE
mysql: optimize code for CERT

### PR DESCRIPTION
**Pull request by bajinsheng**
Within: https://github.com/sqlancer/sqlancer/issues/822

### CODE REVIEW

File: src/sqlancer/mysql/ast/MySQLSelect.java [hunk 1]
```diff
@@ -18,6 +18,10 @@ public void setSelectType(SelectType fromOptions) {
         this.setFromOptions(fromOptions);
     }
 
+    public SelectType getSelectType() {
```
Comment (mrigger) [inline src/sqlancer/mysql/ast/MySQLSelect.java:21]: Isn't this duplicate to the method below?

Comment (bajinsheng) [reply src/sqlancer/mysql/ast/MySQLSelect.java:21]: Aha yes. Will fix later.

**Comment by bajinsheng** [comment]
Seems a bug in jdbc driver?

**Comment by mrigger** [comment]
Yes, seems so, but likely not caused by the change.

**Pull request merged**